src/pages/SupplierInvoicesPage.js

```diff
@@ -100,8 +153,55 @@ export class SupplierInvoicesPage extends GenericPage {
         return invoice.entryDate.toDateString();
       case 'comment':
         return invoice.comment;
+      case 'remove':
+        return {
+          type: 'checkable',
+          icon: 'md-remove-circle',
+          isDisabled: invoice.isFinalised || !invoice.isExternalSI,
+        };
     }
   }
+
+  render() {
+    return (
+      <View style={globalStyles.pageContentContainer}>
+        <View style={globalStyles.container}>
+          <View style={globalStyles.pageTopSectionContainer}>
+            <View style={globalStyles.pageTopLeftSectionContainer}>
+              {this.renderSearchBar()}
+            </View>
+            <View style={globalStyles.pageTopRightSectionContainer}>
+              <PageButton
+                text={buttonStrings.new_supplier_invoice}
+                onPress={() => this.setState({ isCreatingInvoice: true })}
+              />
+            </View>
+          </View>
+          {this.renderDataTable()}
+          <BottomConfirmModal
+            isOpen={this.state.selection.length > 0}
+            questionText={modalStrings.remove_these_items}
+            onCancel={() => this.onDeleteCancel()}
+            onConfirm={() => this.onDeleteConfirm()}
+            confirmText={modalStrings.remove}
+          />
+          <SelectModal
+            isOpen={this.state.isCreatingInvoice}
+            options={this.props.database.objects('Supplier')}
+            placeholderText={modalStrings.start_typing_to_select_supplier}
+            queryString={'name BEGINSWITH[c] $0'}
+            sortByString={'name'}
+            onSelect={name => {
+              this.onNewSupplierInvoice(name);
+              this.setState({ isCreatingInvoice: false });
+            }}
+            onClose={() => this.setState({ isCreatingInvoice: false })}
+            title={modalStrings.search_for_the_customer}
+          />
```

Comment:
Definitely leave for now - it will only need to be refactored when we merge in master anyway